frontend/src/constants/env.ts

```diff
@@ -1,6 +1,3 @@
 export const ENVIRONMENT = {
-	baseURL:
```

Comment:
@srikanthccv please remove this change

Reply:
Yes, I was going to undo this. Just wanted to let others review the work so far.

Reply:
cool ..